git-activity

```diff
@@ -89,7 +89,7 @@ for week_n in $(seq 0 52); do
   if [[ "${current_month}" != "${month_week}" ]]; then
     current_month=$month_week
     weeks_in_month=0
-    printf "%s%s" "$current_month" "$space"
+    printf "%s%s" "${current_month:0:3}" "$space"
```

Comment:
Awesome! This works as expected. There may be another bug if the `current_month` variable is **shorter than 3 characters**, but it's solved by using `%-3s` instead of just `%s` for the interpolation. Like this:

```bash
printf "%-3s%s" "${current_month:0:3}" "$space"
```

Let me know if you want to add that change in your current PR. If some time passes and there's no activity I'll add the change on your PR and merge it anyway (since all of this was your contribution). Thank you for sharing your code :)!

Reply:
No, thank *you* for making this in the first place! This was just a very small fix, not even worth mentioning 
I've added a commit with your requested change
Have a good day!